src/components/markdown-renderer/invalid-yaml-alert.tsx

```diff
@@ -11,14 +11,14 @@ import { InternalLink } from '../common/links/internal-link'
 import { ShowIf } from '../common/show-if/show-if'
 
 export interface InvalidYamlAlertProps {
-  showYamlError: boolean
+  show: boolean
 }
```

Comment:
Can't we just get rid of all these interfaces that only contain a `show` boolean and instead use a common show-modal interface?

Reply:
Merged